src/authors/authors.html

```diff
@@ -1,35 +1,41 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Battleship</title>
+  </head>
 
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-
-<body>
-   <main class="main">
- 
-        <nav class="pop">
-            <h1 class="title">Authors</h1>
-            <ul class="menu">
-             <li> <a class="link--github" href="https://github.com/MarcinRycko" target="_blank">Product Owner: Marcin Rycko</a></li>
-             <li><a class="link--github" href="https://github.com/JustD1d1t" target="_blank">Tech Lead:Dawid Starzyk</a> </li>
-             <li> <a class="link--github" href="https://github.com/alicjaguzik" target="_blank">Development Manager:Alicja Guzik</a></li>
-             <li> <a class="link--github" href="https://github.com/MaciekPon" target="_blank">Maciej Ponicki</a></li>
-             <li><a class="link--github" href="https://github.com/WelcomeInFront" target="_blank">Justyna Rozbicka</a> </li>
-              <li> <a class="link--github" href="#" target="_blank">Dawid Złobecki</a></li>
-             <li> <a class="link--github" href="https://github.com/ruljin" target="_blank">Mentor: Filip Kuca</a></li>
-             
-         </ul>
-           <a class="return" href="index.html">Return</a>
-        </nav>
-        
-           
-        
-        
-        </main>
- </body>
-
-</html>
\ No newline at end of file
+  <body>
+    <main class="main">
+      <nav class="pop">
+        <h1 class="title">Authors</h1>
+        <ul class="menu">
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/MarcinRycko" target="_blank">Product Owner: Marcin Rycko</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/JustD1d1t" target="_blank">Tech Lead: Dawid Starzyk</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/alicjaguzik" target="_blank">Development Manager: Alicja Guzik</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/MaciekPon" target="_blank">Maciej Ponicki</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/WelcomeInFront" target="_blank">Justyna Rozbicka</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/DawidZlobecki" target="_blank">Dawid Złobecki</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/ruljin" target="_blank">Mentor: Filip Kuca</a>
+          </li>
+        </ul>
+        <a class="return" href="index.html">Return</a>
```

Comment:
I think, `Return` should be at the bottom of the right corner like on other pages. Right now it is on the bottom of the container, which is not always height enough to place it on the bottom.